csrankings-j.csv

```diff
@@ -664,9 +663,9 @@ Jeannette Bohg,Stanford University,https://web.stanford.edu/~bohg/,rjnJnEkAAAAJ
 Jed Brown,University of Colorado Boulder,https://www.colorado.edu/cs/users/jeka2967,x_9_NGwAAAAJ
 Jed Crandall,Arizona State University,https://www.cs.unm.edu/~crandall,nOF9akgAAAAJ
 Jed R. Brubaker,University of Colorado Boulder,http://www.colorado.edu/cmci/people/information-science/jed-brubaker,8LEH940AAAAJ
-Jedidiah McClurg,Colorado School of Mines,https://www.jrmcclurg.com,SVuYyQ4AAAAJ
+Jedidiah McClurg,Colorado State University,https://www.jrmcclurg.com,SVuYyQ4AAAAJ

```

Comment:
Web page says "retired from academia"; please remove this and other entry for same faculty member.

Reply:
@emeryberger - I have removed Jed from the file. Apologies for the oversight